Octokit.Tests.Integration/Clients/PullRequestsReviewRequestClientTests.cs

```diff
@@ -0,0 +1,164 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Octokit;
+using Octokit.Tests.Integration;
+using Octokit.Tests.Integration.Helpers;
+using Xunit;
+
+public class PullRequestsReviewRequestClientTests : IDisposable
+{
+    private readonly IGitHubClient _github;
+    private readonly IPullRequestReviewRequestsClient _client;
+    private readonly RepositoryContext _context;
+
+    const string branchName = "new-branch";
+    const string collaboratorLogin = "m-zuber-octokit-integration-tests";
+    const string path = "CONTRIBUTING.md";
```

Comment:
Ditto with this one, if it's not re-used multiple times it doesn't need to be a class level const